.github/workflows/documentation.yml

```diff
@@ -0,0 +1,34 @@
+name: deploy documentation
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Build documentation with doctum
+        run: ./scripts/run_doctum_docs_on_github.sh
+
+      - name: Deploy
+        uses: reggionick/s3-deploy@v3
+        with:
+          folder: doctum-output/build/
+          bucket: rokka-io
```

Comment:
the documentation was using a secret for this as well, but i guess there is no harm in having this here? we disclosed the name in the travis config anyways...

Reply:
Nah, not a big deal, we can leave it like that